DNN Platform/Website/Install/UpgradeWizard.aspx

```diff
@@ -57,15 +57,23 @@
                         </div>
                         </a>
                     </li>
+                    <li><a href="#upgradeSecurity">
+                        <div class="dnnWizardStep">
+                            <span class="dnnWizardStepNumber">2</span>
+                            <span class="dnnWizardStepTitle"><%= LocalizeString("Security")%></span>
+                            <span class="dnnWizardStepArrow"></span>
+                        </div>
+                        </a>
+                    </li>
                     <li><a href="#upgradeInstallation">
                          <div class="dnnWizardStep">
-                            <span class="dnnWizardStepNumber">2</span>
+                            <span class="dnnWizardStepNumber">3</span>
                             <span class="dnnWizardStepTitle"><%= LocalizeString("Upgrade")%></span>
                             <span class="dnnWizardStepArrow"></span>
                         </div>
                         </a>
                      </li>
-                    <li><a href="#upgradeViewWebsite">
+                    <li><a href="#upgradeInstallation">
```

Comment:
Was this supposed to be changed?

Reply:
Yes, that was on purpose. The thing is that the `upgradeViewWebsite` link was pointing to an empty `<div>`, unnecessarily. And there was some Javascript code to handle this case. With this change, when you click on this link it just stays in the installation progress panel.